linodecli/helpers.py

```diff
@@ -27,3 +28,26 @@ def handle_url_overrides(url):
     return parsed_url._replace(
         **{k: v for k, v in overrides.items() if v is not None}
     ).geturl()
+
+
+def filter_markdown_links(text):
+    """
+    Returns the given text with Markdown links converted to human-readable links.
+    """
+
+    result = text
+
+    # Find all Markdown links
+    r = re.compile(r"\[(?P<text>.*?)]\((?P<link>.*?)\)")
```

Comment:
That's a nice regular expression! First time to learn about named group in `re`.